content/governance.md

```diff
@@ -21,3 +21,32 @@ The CWL leadership team consists of the following people, listed in alphabetical
 *   Jeff Gentry, Foundation Medicine, OpenWDL; <https://orcid.org/0000-0001-5351-8442>
 *   Hervé Ménager, Institut Pasteur; <https://orcid.org/0000-0002-7552-1009>
 *   Stian Soiland-Reyes, University of Manchester; <https://orcid.org/0000-0001-9842-9718>
+
+## CWL Technical Team
+
+The team is initially responsible for approving new versions of the CWL standards before that proposal goes to the CWL Leadership Team for the final vote. The team consists of the following people, listed in alphabetical order by their last name:
+
+* Fabrice Brito, Terradue
+  * <https://github.com/fabricebrito>
+  * <https://matrix.to/#/@fabrito:matrix.org>
+* Iacopo Colonnelli, University of Torino, Computer Science Dept.
+  * <https://github.com/glassofwhiskey>
+  * <https://matrix.to/#/@glassofwhiskey:matrix.org>
+  * <https://orcid.org/0000-0001-9290-2017>
+* Michael R. Crusoe, ELIXIR-NL
+  * <https://github.com/mr-c>
+  * <https://matrix.to/#/@mr-c:matrix.org>
+  * <https://orcid.org/0000-0002-2961-9670>
+* Bogdan Gavrilović, Seven Bridges
+  * <https://github.com/bogdang989>
+  * <https://matrix.to/#/@bogdan_gavrilovic:matrix.org>
+  * <https://orcid.org/0000-0003-1550-1716>
+  * <https://github.com/orgs/common-workflow-language/teams/tech_team>
+  * <https://matrix.to/#/#cwltool-dev:matrix.org>
```

Comment:
These links should go in the explanation paragraph on line 27

Reply:
I just added the GitHub link to the paragraph on line 27 (https://github.com/common-workflow-language/cwl-website/pull/117/commits/34f2667da3ad0687a34d42243ac3d941f4640822) , but wasn't sure how to work in the Matrix link. Probably better if someone else writes that in.